src/main/java/sirius/biz/jobs/batch/file/LineBasedImportExportJob.java

```diff
@@ -0,0 +1,93 @@
+import sirius.biz.storage.layer3.FileParameter;
+import sirius.kernel.commons.Values;
+
+import java.io.IOException;
+
+/**
+ * Provides a job for transforming a line based file into another.
+ * <p>
+ * This is implemented by a {@link LineBasedImportExportJobFactory} and will reaa line based file, process a row
+ * and output one or more rows into an output file.
+ */
+public abstract class LineBasedImportExportJob extends LineBasedImportJob {
+
+    protected final LineBasedExportJob exportJob;
+
+    /**
+     * Creates a new job for the given factory, name and process.
+     *
+     * @param inputFileParameter   the parameter which is used to derive the import file from
+     * @param destinationParameter the parameter used to select the destination for the file being written
+     * @param fileTypeParameter    the file type to use when writing the line based data
+     * @param process              the process context itself
+     */
+    protected LineBasedImportExportJob(FileParameter inputFileParameter,
+                                       FileOrDirectoryParameter destinationParameter,
+                                       EnumParameter<ExportFileType> fileTypeParameter,
+                                       ProcessContext process) {
+        super(inputFileParameter, process);
+        this.exportJob = new LineBasedExportJob(destinationParameter, fileTypeParameter, process) {
+            @Override
+            protected void executeIntoExport() throws Exception {
+                // ignored
+            }
+
+            @Override
+            protected String determineFilenameWithoutExtension() {
+                return LineBasedImportExportJob.this.determineFilenameWithoutExtension();
+            }
+        };
+    }
+
+    /**
+     * Determines the base name to use for the file.
+     * <p>
+     * This will be expanded by the date and also by additional suffixes to generate a unique name. Also the file
+     * extension as supplied by {@link FileExportJob#determineFileExtension()} will be appended.
+     *
+     * @return the base file name to use
+     */
+    protected abstract String determineFilenameWithoutExtension();
+
+    @Override
+    public void execute() throws Exception {
+        exportJob.execute();
+        super.execute();
+    }
+
+    @Override
+    public void handleRow(int lineNumber, Values row) {
+        handleRow(lineNumber, row, exportJob.export);
+    }
+
+    /**
+     * Invoked for each line in the given input file.
+     * <p>
+     * The provided export writes into the selected output file.
+     * </p>
+     *
+     * @param lineNumber the line of the import being processed
+     * @param row        the data in this line
+     * @param export     the export to write to
+     */
+    protected abstract void handleRow(int lineNumber, Values row, LineBasedExport export);
+
+    @Override
+    public void close() throws IOException {
+        super.close();
+        exportJob.close();
```

Comment:
Wenn super.close ne Exception wirft, wird der Export-Job nicht geschlossen :(
```suggestion
        try (Closeable c = super) {
            exportJob.close();
        }
```
Das try with resources macht intern ziemlich viele coole Sachen: https://www.oracle.com/technical-resources/articles/java/trywithresources.html

Reply:
```suggestion
        try(Closeable c1 = super, Closeable c2 = exportJob) {}
```
geht natürlich auch, falls es compiliert